panoptes_client/subject.py

```diff
@@ -214,6 +215,9 @@ def set_raw(self, raw, etag=None, loaded=True):
         elif loaded:
             self._original_metadata = None
 
+    def status_in_workflow(self, workflow_id):
+        return list(SubjectWorkflowStatus.where(subject_id=self.id, workflow_id=workflow_id))
```

Comment:
Instead of returning an explicit list i think we should re-use the existing paradigm of using the next method to return a page of resource data, e.g. 
```
 return next(SubjectWorkflowStatus.where(subject_id=self.id))
```
https://github.com/zooniverse/panoptes-python-client/blob/b25c3b88ce29ae0c19597119b456c799f9d41426/panoptes_client/subject_workflow_status.py#L12-L15

I think consistency of the returned API resource response objects is key here. That said if list provides the same interface (I don't think it does) then that's ok too.

Reply:
Got it. Yeah you are right list does not provide the same interface. i'll  update. Thanks~